database/models.py

```diff
@@ -27,3 +29,54 @@ class PlanRegulationGroup(VersionedBase):
 
     short_name: Mapped[unique_str]
     name: Mapped[language_str]
+    # värikoodi?
+    # group_type: oma koodilista
+
+
+class PlanRegulation(PlanBase):
+    """
+    Kaavamääräys
+    """
+
+    __tablename__ = "plan_regulation"
+
+    plan_regulation_group_id: Mapped[uuid.UUID] = mapped_column(
+        ForeignKey(
+            "hame.plan_regulation_group.id", name="plan_regulation_group_id_fkey"
+        )
+    )
+
+    type_of_plan_regulation_id: Mapped[uuid.UUID] = mapped_column(
+        ForeignKey(
+            "codes.type_of_plan_regulation.id", name="type_of_plan_regulation_id_fkey"
+        )
+    )
+    type_of_verbal_plan_regulation_id: Mapped[uuid.UUID] = mapped_column(
```

Comment:
Oh, one more thing. I'm not quite sure how sanallinen määräys is supposed to work. But probably it works so that type of verbal plan regulation is *only* defined for those regulations that have type of plan regulation set as verbal.

This means we also have to allow nullable=True for type_of_verbal_plan_regulation_id column.